app/views/govuk_component/taxonomy_sidebar.raw.html.erb

```diff
@@ -28,7 +28,7 @@
         </p>
 
         <% item[:related_content] ||= [] %>
-        <% if item[:related_content].any? && item_index < 2 %>
+        <% if item[:related_content].any? && (item_index < 2 || item[:is_curated]) %>
```

Comment:
I think this logic should be moved to the navigation helpers. I've attempted to do that in the past, but the PR wasn't merged because we weren't ready to do this. Here is how it looked:

- https://github.com/alphagov/govuk_navigation_helpers/pull/68/commits/a02334378abd7003aa76e4fd307c504bcf5de263

What do you think?